cmd/cerbos/compile/compile.go

```diff
@@ -50,15 +50,16 @@ cerbos compile --skip-tests /path/to/policy/repo
 `
 
 type Cmd struct { //nolint:govet // Kong prints fields in order, so we don't want to reorder fields to save bytes.
-	Dir           string               `help:"Policy directory" arg:"" required:"" type:"existingdir"`
-	IgnoreSchemas bool                 `help:"Ignore schemas during compilation"`
-	Tests         string               `help:"Path to the directory containing tests. Defaults to policy directory." type:"existingdir"`
-	RunRegex      string               `help:"Run only tests that match this regex" name:"run"`
-	SkipTests     bool                 `help:"Skip tests"`
-	Output        flagset.OutputFormat `help:"Output format (${enum})" default:"tree" enum:"tree,list,json" short:"o"`
-	Color         *outputcolor.Level   `help:"Output color level (auto,never,always,256,16m). Defaults to auto." xor:"color"`
-	NoColor       bool                 `help:"Disable colored output" xor:"color"`
-	Verbose       bool                 `help:"Verbose output on test failure"`
+	Dir           string                           `help:"Policy directory" arg:"" required:"" type:"existingdir"`
+	IgnoreSchemas bool                             `help:"Ignore schemas during compilation"`
+	Tests         string                           `help:"Path to the directory containing tests. Defaults to policy directory." type:"existingdir"`
+	RunRegex      string                           `help:"Run only tests that match this regex" name:"run"`
+	SkipTests     bool                             `help:"Skip tests"`
+	Output        flagset.OutputFormat             `help:"Output format (${enum})" default:"tree" enum:"tree,list,json" short:"o"`
+	TestOutput    flagset.VerificationOutputFormat `help:"Test output format (${enum})" default:"tree" enum:"tree,list,json,junit"`
```

Comment:
Don't set a default here but mark the flag as optional. If it's not explicitly set by the user, we should set its value to the value of the `Output` flag. You can do that manually in the `main` function or through a [Resolver](https://github.com/alecthomas/kong/blob/master/README.md#resolver---support-for-default-values-from-external-sources)

Reply:
Unfortunately, it's impossible to set `enum` fields as `optional`. I introduced an `unspecified` enum value to solve this, though I am not sure if there is a better way.

Reply:
Could the flag be made into a pointer so that if it's not defined, it'll be nil?
 
I suppose you could also make it a plain string field and have a [Validator](https://github.com/alecthomas/kong/blob/master/README.md#validation) that checks to make sure it contains only the supported values.